protocol/src/main/java/io/camunda/zeebe/protocol/record/value/ProcessInstanceModificationRecordValue.java

```diff
@@ -76,6 +77,9 @@ interface ProcessInstanceModificationActivateInstructionValue {
 
     /** Returns a list of variable instructions (if available), or an empty list. */
     List<ProcessInstanceModificationVariableInstructionValue> getVariableInstructions();
+
+    /** Returns all ancestor scope keys of the element that will be activated. */
```

Comment:
:wrench: We could mention that the property is set only in the event.

For example:

```suggestion
    /**
     * Returns all ancestor scope keys of the element that was activated. The property is set only
     * in the event after the modification is applied.
     */
```